ui/pom.xml

```diff
@@ -0,0 +1,129 @@
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public
+ * License along with this software; if not, write to the Free
+ * Software Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA
+ * 02110-1301 USA, or see the FSF site: http://www.fsf.org.
+-->
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+  <parent>
+    <groupId>com.xwiki.googleapps</groupId>
+	<artifactId>application-googleapps-parent</artifactId>
+	<version>2.5-SNAPSHOT</version>
+  </parent>
+  <artifactId>application-googleapps</artifactId>
+  <packaging>xar</packaging>
+  <name>Google Apps Integration (Pro) UI</name>
+  <description>This extension allows to connect Google Apps to XWiki</description>
+  <properties>
+    <!-- The list of documents that have an implicit unlimited free license. The users can view these documents without
+      buying a license or getting a trial license, but they cannot edit or delete them. -->
+    <xwiki.extension.licensing.publicDocuments>
+      GoogleApps.WebHome
+    </xwiki.extension.licensing.publicDocuments>
+    <xwiki.extension.licensing.excludedDocuments>
+      GoogleApps.GoogleAppsConfig,
+      GoogleApps.WebPreferences
+    </xwiki.extension.licensing.excludedDocuments>
+    <!-- Name to display by the Extension Manager -->
+    <xwiki.extension.name>Google Apps Integration (Pro)</xwiki.extension.name>
+    <xwiki.extension.category>application</xwiki.extension.category>
+    <!-- Don't run CLIRR here since there's no Java code. -->
+    <xwiki.clirr.skip>true</xwiki.clirr.skip>
+    <!-- Don't run enforcer here since there's no Java code to avoid java7 requirement. -->
+    <xwiki.enforcer.skip>true</xwiki.enforcer.skip>
+  </properties>
+  <dependencies>
+    <dependency>
+      <groupId>${project.groupId}</groupId>
+      <artifactId>application-googleapps-api</artifactId>
+      <version>${project.version}</version>
+      <scope>runtime</scope>
+    </dependency>
+    <dependency>
+      <groupId>com.xwiki.licensing</groupId>
+      <artifactId>application-licensing-licensor-api</artifactId>
+      <version>${licensing.version}</version>
+    </dependency>
+  </dependencies>
+  <issueManagement>
+    <system>GitHub</system>
+    <url>https://github.com/xwikisas/application-googleapps/issues</url>
+  </issueManagement>
+  <scm>
+    <connection>scm:git:git://github.com/xwikisas/application-googleapps.git</connection>
+    <developerConnection>scm:git:git@github.com:xwikisas/application-googleapps.git</developerConnection>
+    <url>https://github.com/xwikisas/application-googleapps</url>
+    <tag>HEAD</tag>
+  </scm>
```

Comment:
`issueManagement` and `scm` tags should be defined only in the parent `pom.xml` file of the project.

Reply:
@polx why did you resolve this conversation?

Reply:
Thanks, this was an oversight in the big PR.
I'm now going more systematically and have solved this in 82e630b3d607a0aa024c2812efae6879bb19109e.
thanks
paul